lib/dmv.rb

```diff
@@ -1,5 +1,5 @@
 class Dmv
-
+  attr_reader :facilities
   def initialize
     @facilities = []
```

Comment:
Were you able to make this bug pass? We needed to change line 12 of this class to a different enumerable to get this one working.